daemon/container_operations.go

```diff
@@ -54,7 +54,8 @@ func (daemon *Daemon) buildSandboxOptions(cfg *config.Config, container *contain
 		sboxOptions = append(sboxOptions, libnetwork.OptionUseExternalKey())
 	}
 
-	if err := setupPathsAndSandboxOptions(container, cfg, &sboxOptions); err != nil {
+	// Add OS-specific Sandbox options.
+	if err := buildSandboxOptionsOS(container, cfg, &sboxOptions); err != nil {
```

Comment:
More bike-shedding; I know we have some other "options" / platform-specific code for which we use `platform` as term for this, e.g. `ValidatePlatformConfig` and `setPlatformDefaults` in the daemon/config package. Perhaps it would be clear to follow that convention.

Reply:
Oh, yes - I'll change it.